finance_manager/pom.xml

```diff
@@ -132,6 +132,8 @@
 			<artifactId>rest-api</artifactId>
 			<version>0.0.1-SNAPSHOT</version>
 		</dependency>
+
```

Comment:
no issues, but you dont need extra lines

Reply:
Deleted